libcnb-cargo/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "libcnb-cargo"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.56"
+license = "BSD-3-Clause"
+description = "Cargo command for managing buildpacks written with libcnb.rs"
+repository = "https://github.com/Malax/libcnb.rs"
+readme = "../README.md"
```

Comment:
Should `libcnb-cargo` have a different README than to `libcnb`? eg: One at `libcnb-cargo/README.md` that's only a few lines long and instead redirects to the `libcnb` crate? Or else none at all for the moment?

Having the full libcnb project readme for a sub-crate seems like it might be confusing?

Reply:
Added in 8226d4f